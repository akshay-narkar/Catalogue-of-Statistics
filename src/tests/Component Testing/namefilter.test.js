import renderer from 'react-test-renderer';
import Namefilter from '../../Components/namefilter';

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<Namefilter />).toJSON();

  expect(elem).toMatchSnapshot();
});
