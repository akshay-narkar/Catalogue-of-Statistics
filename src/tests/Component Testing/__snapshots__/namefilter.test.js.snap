// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<div
  className="ms-5 my-3"
>
  <label
    className="form-label d-flex align-items-baseline"
    htmlFor="exampleFormControlInput1"
  >
    <p
      className="m-0"
    >
      Team Name:
    </p>
    <input
      className="form-control text-box ms-2 w-50"
      id="exampleFormControlInput1"
      onChange={[Function]}
      placeholder="Title"
      type="text"
      value=""
    />
  </label>
</div>
`;
